tests/e2e/Services/Database/DatabaseCustomServerTest.php

```diff
@@ -821,7 +821,7 @@ public function testIndexLimitException()
         $this->assertIsArray($collection['body']['attributes']);
         $this->assertIsArray($collection['body']['indexes']);
         $this->assertCount(64, $collection['body']['attributes']);
-        $this->assertCount(61, $collection['body']['indexes']);
+        $this->assertCount(59, $collection['body']['indexes']);
```

Comment:
Let's improve our existing database tests. There we never had createAt/updateAt. Let's make tests where we check createdAt is created properly, and updateAt updated when updating document.

Reply:
Talking about that, lets also add tests to ensure people cannot touch createdAt and updatedAt. So we can present this as server-managed (secure) info.